### ISSUE TITLE
Several emojis not being converted to unicode

### ISSUE THREAD

**Issue by Rulikkk** [wooorm/retext-emoji#15]
- Speaking_head does not get converted to emoji. I see just `:speaking_head:` instead of 🗣 
- No_entry produces the following error, if used ⛔️ :
```
TypeError: Cannot read property 'names' of undefined
    at visitor (retext-emoji/index.js:109:22)
```

**Comment by wooorm** [comment]
PRs welcome!

**Comment by omaijdot** [comment]
i fixed this issue already. it is due to i have compression svg file, then found out that Safari Browser is not able decode it properly. so that i exclude to do compression if using Safari Browser.
https://github.com/wooorm/retext-emoji/issues/16

**Comment by wooorm** [comment]
Closed by 321e8948b7f8498917e595f28d8bdb85c5233d9f.
